packages/components/pagination/src/page-navigator/page-navigator.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
```

Comment:
Why not TS first?

Reply:
There are dependent packages that we need to migrate first.